x-pack/test/api_integration/apis/index.ts

```diff
@@ -11,30 +11,30 @@ export default function ({ loadTestFile }: FtrProviderContext) {
   describe('apis', function () {
     this.tags('ciGroup6');
 
-    loadTestFile(require.resolve('./search'));
-    loadTestFile(require.resolve('./es'));
-    loadTestFile(require.resolve('./security'));
-    loadTestFile(require.resolve('./spaces'));
-    loadTestFile(require.resolve('./monitoring'));
-    loadTestFile(require.resolve('./xpack_legacy'));
-    loadTestFile(require.resolve('./features'));
-    loadTestFile(require.resolve('./telemetry'));
-    loadTestFile(require.resolve('./logstash'));
-    loadTestFile(require.resolve('./kibana'));
-    loadTestFile(require.resolve('./metrics_ui'));
-    loadTestFile(require.resolve('./beats'));
-    loadTestFile(require.resolve('./console'));
-    loadTestFile(require.resolve('./management'));
-    loadTestFile(require.resolve('./uptime'));
-    loadTestFile(require.resolve('./maps'));
+    // loadTestFile(require.resolve('./search'));
+    // loadTestFile(require.resolve('./es'));
+    // loadTestFile(require.resolve('./security'));
+    // loadTestFile(require.resolve('./spaces'));
+    // loadTestFile(require.resolve('./monitoring'));
+    // loadTestFile(require.resolve('./xpack_legacy'));
+    // loadTestFile(require.resolve('./features'));
+    // loadTestFile(require.resolve('./telemetry'));
+    // loadTestFile(require.resolve('./logstash'));
+    // loadTestFile(require.resolve('./kibana'));
+    // loadTestFile(require.resolve('./metrics_ui'));
+    // loadTestFile(require.resolve('./beats'));
+    // loadTestFile(require.resolve('./console'));
+    // loadTestFile(require.resolve('./management'));
+    // loadTestFile(require.resolve('./uptime'));
+    // loadTestFile(require.resolve('./maps'));
```

Comment:
Are these intentional along with the test fixes below or leftovers from testing?